lib/error_tracker.ex

```diff
@@ -383,4 +382,13 @@ defmodule ErrorTracker do
     Telemetry.new_occurrence(occurrence, muted)
     occurrence
   end
+
+  @doc false
+  def __default_json_encoder__ do
+    # Elixir 1.18+ includes the JSON module. On older versions we should fall back to Jason (which
+    # is listed as an optional dependency).
+    if Version.match?(System.version(), ">= 1.18.0"),
+      do: JSON,
+      else: Jason
```

Comment:
👀 

I think this may generate compile warnings. I tried compiling in v1.180 and simulating Jason is not available (changing it to something different, like `Jason2`) and it generates a compile warning even not being used in runtime.

And that may happen also in the other direction (trying to find `JSON` in Elixir < v1.17, but haven't tested that).

Reply:
This is interesting. I will create two plain Phoenix projects (one with Jason and other without) and see how it behaves.

If this throws warnings we may need to ask in the forum how to handle it properly.